prefect_airbyte/client.py

```diff
@@ -172,17 +172,18 @@ async def trigger_manual_sync_connection(
 
             raise err.AirbyteServerNotHealthyException() from e
 
-    async def get_job_status(self, job_id: str) -> Tuple[str, str, str]:
+    async def get_job_status(
+        self, job_id: str, stream_logs: bool = False
+    ) -> Tuple[str, str, str]:
         """
         Gets the status of an Airbyte connection sync job.
 
         Args:
             job_id: ID of the Airbyte job to check.
+            stream_logs: whether to capture Airbyte job logs in metadata.
```

Comment:
Is there a downside if default `stream_logs=True`?

```suggestion
            stream_logs: Whether to capture Airbyte job logs in metadata.
```

Reply:
Airbyte logs are super verbose so as a user I wouldn't want to see them by default, but totally open to changing this if we think its appropriate